schema/me/conversation/invoice.js

```diff
@@ -0,0 +1,44 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLEnumType } from "graphql"
+import { amount } from "../../fields/money"
+
+const InvoiceState = new GraphQLEnumType({
+  name: "InvoiceState",
+  values: {
+    UNPAID: {
+      value: "unpaid",
+    },
+    PAID: {
+      value: "paid",
+    },
+    VOID: {
+      value: "void",
+    },
+    REFUNDED: {
+      value: "refunded",
+    },
+  },
```

Comment:
I'm a stranger to this `values: { X: { value: "x" } }` structure but it's beautiful in an inception sort of way.